Hyperlint Automation: Meta Description Fixes (#20437)
cloudflare/cloudflare-docs: src/content/docs/rules/transform/request-header-modification/index.mdx

head:
  - tag: title
    content: HTTP request header modification rules

>   description: Learn to modify HTTP request headers with Cloudflare's rules. Set, overwrite, or remove headers via dashboard or API. Understand limitations and best practices.

Reviewer: ```suggestion
description: Learn how to modify HTTP request headers with Cloudflare's rules.
```

	file="snippets-alternative"
	params={{ suffix: "request header modifications" }}
/>

> ## Important remarks

Reviewer: ```suggestion

## Important remarks
```
